sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/models/ErrorContext.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
There is a class in azure-core-amqp called ErrorContext which is about errors that occur on the AMQP link, session, or connection. Will this conflict with that? There is also a SessionErrorContext, etc.

Reply:
Renamed this to `EventProcessingErrorContext`.